lib/WP/Functions.php

```diff
@@ -24,6 +24,34 @@ function wpRemoteRetrieveResponseMessage() {
     return call_user_func_array('wp_remote_retrieve_response_message', func_get_args());
   }
 
+  function addFilter() {
```

Comment:
I don't understand why would we want to merge all of the functions inside one big class. Why don't we separate it by usage?

Reply:
I was thinking about having all WP functions in one class to make it easier to mock them. Having a large class is not an issue in that case because all methods are basically the same. I had also the idea of using `__call` to generalize the implementation but this will hurt the performance!